src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const rentalCost = days * COST_PER_DAY;
+  let discount = 0;
+
+  if (SHORT_TERM <= days) {
+    discount = SHORT_TERM_DISCOUNT;
+  }
+
+  if (LONG_TERM <= days) {
+    discount = LONG_TERM_DISCOUNT;
+  }
```

Comment:
The current discount logic overrides the $20 discount with a $50 discount for rentals of 7 days or more. To align with the intended discount structure, you should adjust the logic to ensure that both discounts are applied cumulatively if applicable. Consider checking for the longer-term discount first and then adding the shorter-term discount if the conditions are met.